ui/src/components/ExportFab.js

```diff
@@ -5,16 +5,25 @@ import "./ExportFab.css";
 import CloudUpload from "@material-ui/icons/CloudUpload";
 import Button from "@material-ui/core/Button";
 import Tooltip from "@material-ui/core/Tooltip";
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
 import React from "react";
 
 class ExportFab extends React.Component {
   constructor(props) {
     super(props);
-    this.handleClick = this.handleClick.bind(this);
+    this.state = {open: false};
+    this.handleClick = this.handleClick.bind(this);     
+    this.handleSave = this.handleSave.bind(this);    
   }
 
   render() {
     return (
+      <div>
       <Tooltip title="Export to Saturn">
```

Comment:
Can you fix the formatting? Eg this line should be indented two spaces compared to previous line. Also, this I think this div should end on line 37 instead of line 63.

Reply:
There should be an outer <div> enclosing all the tags, I got an error without it.